src/WebJobs.Script/Description/Node/NodeFunctionInvoker.cs

```diff
@@ -367,6 +354,34 @@ protected override void OnScriptFileChanged(object sender, FileSystemEventArgs e
             return context;
         }
 
+        private static Dictionary<string, object> NormalizeBindingData(Dictionary<string, object> bindingData)
+        {
+            Dictionary<string, object> normalizedBindingData = new Dictionary<string, object>();
+
+            foreach (var pair in bindingData)
+            {
+                var name = pair.Key;
+                var value = pair.Value;
+                if (value != null && !IsEdgeSupportedType(value.GetType()))
+                {
+                    // we must convert values to types supported by Edge
+                    // marshalling as needed
+                    value = value.ToString();
+                }
+
+                // "camel case" the normally Pascal cased properties by
+                // converting the first letter to lower if needed
+                // While for binding purposes case doesn't matter,
+                // we want to normalize the case to something Node
+                // users would expect to reference in code (e.g. "dequeueCount" not "DequeueCount")
+                name = Utility.ToLowerFirstCharacter(name);
```

Comment:
I also considered a solution where I wrapped JS objects using a Proxy to allow case insensitive property access, but I think that would just lead to confusion.